### PR TITLE
uroot_test: add test for building with gbb and GO111MODULE=off

### PR DESCRIPTION
**Pull request by rminnich**
Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

File: uroot_test.go [hunk 1]
```diff
@@ -282,7 +282,12 @@ func TestUrootCmdline(t *testing.T) {
 		gbbTest.args = append([]string{"-build=gbb"}, gbbTest.args...)
 		gbbTest.env = append(gbbTest.env, "GO111MODULE=on")
 
-		bbTests = append(bbTests, gbbTest, bbTest)
+		gbbNoModuleTest := test
+		gbbTest.name = gbbTest.name + " gbbNoModules"
```
Comment (hugelgupf) [inline uroot_test.go:286]: s/gbbTest/gbbNoModuleTest.name =

s/gbbTest.args/gbbNoModuleTest.args

s/gbbTest.env/gbbNoModuleTest.env

**Review by hugelgupf — DISMISSED** [review]

File: uroot_test.go (hunk 1 above)
Comment (rminnich) [reply uroot_test.go]: done, thanks.

**Comment by rminnich** [comment]
I mean the dismissing part. Is this more gerrit-like?

**Review by hugelgupf — APPROVED** [review]

**Comment by rminnich** [comment]
ah geez. 15m -- I thought I'd fixed that timeout. I'll go look again.

**Comment by rminnich** [comment]
we kinda realized we don't need this, since -build=bb really now means -build=gbb anyway?

**Pull request closed without merging**